### PR TITLE
Toast when session starts/stops

### LINKED ISSUE #1093: SessionRecorder: Toast when session recording starts and stops.
Show a Toast when a session starts recording, and when a session stops recording, so the user knows it happened.

### PR DESCRIPTION
**Pull request by brentpicasso**
resolves #1093

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -529,6 +531,8 @@ def _restart_comms(self):
         self._rc_api.shutdown_api()
         self.init_rc_comms()
 
+    def _on_session_recording(self, instance, is_recording):
+        toast('Analysis session started' if is_recording else 'Analysis session stopped', length_long=True)
```
Comment (ryandoherty) [inline main.py]: The messaging here seems unclear to me. What is an 'analysis session'? I'd go with something like 'recording' or 'logging' session instead, seems clearer to me.

**Comment by ryandoherty** [comment]
Looks good other than wordsmithing!

File: main.py (hunk 1 above)
Comment (brentpicasso) [inline main.py]: Logging might be a good one. I don't want to confuse that with RCP SD card logging, though.

I'll try "Recording Session' or similar.

**Comment by ryandoherty** [comment]
Soooooooooooooo I hate to be that guy, but I think wording that explicitly says 'recording' would make it very self-explanatory. I don't think users will understand that 'Session started' and 'Session ended' means the app is recording. We could make it 'Session recording started' and 'Session recording ended', which to me seems much clearer. I do know that we have to make sure it's clear it isn't SD card recording though.

**Comment by ryandoherty** [comment]
:shipit:

**Pull request merged**